### ISSUE TITLE
[Platform UI k8s] CPU/Disk metrics are not visible on overview page as well as Containers section on Metrics page on 2.6.0.0-b30 build universe

### ISSUE THREAD

**Issue by RP1807** [yugabyte/yugabyte-db#8976]
Observed this issue on the platform created using openshift helm-based installation.

1. Create a universe with 2.6.0.0-b30 build
2. Started YSQL sample apps
3. Observed that CPU usage and Persistent Volume stats metrics are missing
![Screenshot 2021-06-18 at 11 35 46 AM](https://user-images.githubusercontent.com/24388219/122514283-55ffdc00-d029-11eb-9661-7fa45c073e7d.png)

**Comment by streddy-yb** [comment]
cc @iSignal

**Comment by bhavin192** [comment]
@RP1807 Did you add `rbac.enabled=true` while doing Helm install as given in the docs?

> Note that if you are executing the preceding command as an admin user, then you can set rabc.create=true. Alternatively, you can ask the cluster administrator to perform the next step.

https://docs.yugabyte.com/latest/yugabyte-platform/install-yugabyte-platform/install-software/openshift/#creating-an-instance-of-yugabyte-platform-via-helm

**Comment by RP1807** [comment]
@bhavin192 I set `rbac.enabled=false` but did the optional step to create ClusterRoleBinding with the admin user.

**Comment by bhavin192** [comment]
@RP1807 are you sure that the creation of ClusterRoleBinding was successful? Because when I try to run the given command from docs it gives me this error

```
Error from server: error when creating "STDIN": unsupported role reference kind: ""
```
I fixed the above issue and created universe of 2.6.0.0-b30 and I can see the metrics there.

![image](https://user-images.githubusercontent.com/5154532/122882830-35ea5880-d35a-11eb-83f1-9b791af9e124.png)

**Comment by RP1807** [comment]
@bhavin192 yes it was successful I too faced the same error but after fixing that it was successful. Let me try this again will update here

**Comment by RP1807** [comment]
@bhavin192 I tried this today. I set `rbac.enabled=true` while doing Helm install as I was executing commands with admin user and also did optional ClusterRoleBinding step as mentioned in the docs here.   
https://docs.yugabyte.com/latest/yugabyte-platform/install-yugabyte-platform/install-software/openshift/#helm-based-installation

After following these steps,  I could see the CPU/Disk metrics on the Universe overview page as well as on the Metrics page. I had to refresh the page couple of times though. 

![Screenshot 2021-06-23 at 10 22 44 PM](https://user-images.githubusercontent.com/24388219/123137550-b0f26280-d471-11eb-81de-af70ae27c32c.png)

![Screenshot 2021-06-23 at 10 26 55 PM](https://user-images.githubusercontent.com/24388219/123138501-b7350e80-d472-11eb-9179-27788deb6174.png)

cc @VijiYB
